package.json

```diff
@@ -149,47 +148,8 @@
     "not op_mini all"
   ],
   "jest": {
-    "collectCoverageFrom": [
-      "src/**/*.{js,jsx,ts,tsx}",
-      "!src/{App,Routes,index,registerServiceWorker}.js",
-      "!src/**/*.Component.js",
-      "!src/_playground/**"
```

Comment:
I thought this section was at least in use for what files to gather coverage from (or not gather coverage from), however, it seems to ignore the proper files without this code.  I am curious as to how it knows which files to ignore though.  I am wondering specifically about `*.Component.js` files and anything in the `_playground` directory.

Reply:
looking here:

https://coveralls.io/builds/21661724

it does looks like it picked up the `config` directory. let me see if i can centralize this into `/config/test/jest/config.js`